src/umpire/util/Macros.hpp

```diff
@@ -86,6 +88,19 @@
 
 #define UMPIRE_USE_VAR(x) static_cast<void>(x)
 
+#ifdef UMPIRE_ENABLE_BACKTRACE
+#define UMPIRE_ERROR( msg )                                        \
+{                                                                  \
+  UMPIRE_LOG(Error, msg                                            \
+      << std::endl << umpire::util::BackTrace::Backtrace());       \
+  std::ostringstream umpire_oss_error;                             \
+  umpire_oss_error << " " << __func__ << " "                       \
```

Comment:
I think having the whole backtrace string in the Exception might be a little much.

Reply:
Perhaps.

The advantage of placing the string in the exception is that it will prominantly be visable during the exception handling.

This is only on when someone builds with -DENABLE_BACKTRACE.

Reply:
Okay, makes sense to me 👍